app/components/daisy/data_display/avatar_component.rb

```diff
@@ -0,0 +1,34 @@
+# This is the avatar component.
```

Comment:
At some point, we'll talk about how best to document these classes. But for now, we can keep it simple like this.